core/src/main/scala/org/apache/spark/SparkContext.scala

```diff
@@ -2788,7 +2788,7 @@ object SparkContext extends Logging {
       case "local" =>
         checkResourcesPerTask(1)
         val scheduler = new TaskSchedulerImpl(sc, MAX_LOCAL_TASK_FAILURES, isLocal = true)
-        val backend = new LocalSchedulerBackend(sc.getConf, scheduler, 1)
+        val backend = new LocalSchedulerBackend(sc.getConf, scheduler, 1, sc._resources)
```

Comment:
`sc._resources` is for `driver` rather than `executor`?

Reply:
> sc._resources is for driver rather than executor?
Yes. But in local mode, `driver` is behave as `executor` and `spark.executor.resource.*` doesn't affect.

Reply:
I see. But we also need to pass `_resources` into `LocalEndpoint` and then to `Executor`, not only UI?

Reply:
Maybe we should do `parseOrFindResources` for `executor` in `LocalSchedulerBackend` and then we could get the real resources which configured from `executor`.

Reply:
Ah, right. I'll also fix it. Thanks!